README_CN.md

```diff
@@ -0,0 +1,70 @@
+![LucidGloves](https://cdn.discordapp.com/attachments/785135646082990120/873666310855360582/LucidGlovesBlackwbkgnd.png)
+
+**[English](https://github.com/RyanPiao/lucidgloves/blob/main/README.md) | [简体中文](https://github.com/RyanPiao/lucidgloves/blob/main/README_CN.md)**
```

Comment:
These also need to be updated to the correct repo links (I know the README.CN file doesn't exist here yet)